cmds/ocm/pkg/utils/handling.go

```diff
@@ -45,6 +46,8 @@ func StringElemSpecs(args ...string) []ElemSpec {
 }
 
 func ElemSpecs(list interface{}) []ElemSpec {
+	// This is acceptable as the calling chain will just take a nil slice.
+	defer panics.HandlePanic()
 	if list == nil {
```

Comment:
see comment below concerning panic defers in functions with a panic.

Basically we should catch this problem up the call tree, where the processing chain is executed and an error return is possible. At that place it should be converted to an error,

A good place would be the initial main command execution, the command is aborted because of a severe programming error, and this error is returned to the use as regular error. A panic defer at that place would be a good idea anyway to prevent callstacks to reach the CLI user. But then we would need an option, to enable the callstack again, to give the developer a chance to figure out the concrete location.

Reply:
But the initial command execution is too far above, right? If this util function could be called from the outside by something that imports OCM we should protect that calling library. Not?

Reply:
Actually, I can catch this in the Command error Run.